docs/source/installation/latex.rst

```diff
@@ -115,3 +73,10 @@ your new TeXLive installation:
 
 If you are in a production setup, reload uWSGI using
 ``touch /opt/indico/web/indico.wsgi`` to reload the config file.
+
+As security-related updates are released frequently, it is also
+a good idea to periodically update the TeXLive packages by running:
+
+.. code-block:: shell
+
+    /opt/texlive/bin/x86_64-linux/tlmgr update --all
```

Comment:
It may be better to add `--self` to this `tlmgr` command to update `tlmgr` itself.